lib/plugin.server.js

```diff
@@ -0,0 +1,13 @@
+/** @type {import('@nuxt/types').Module} */
+export default function (ctx, inject) {
+  const sentry = process.sentry || {}
+  // Inject Sentry to the context as $sentry
+  inject('sentry', sentry)
+  ctx.$sentry = sentry
+
+  <% if (options.lazy) { %>
+  const sentryReady = () => Promise.resolve(sentry)
+  inject('sentryReady', sentryReady)
+  ctx.$sentryReady = sentryReady
+  <% } %>
+}
```

Comment:
Would it make sense to also mock `sentryLoad` function?
We could expect people to only call it from the client but it probably wouldn't hurt to also handle it on the server.

Reply:
Im not a big fan of mocking functions that shouldnt be called anyway as it will only allow users to write faulty code and then hide that from them.

Can we think of a use-case where you'd actually want to use sentryLoad on a server? If not I think we could just omit it

Reply:
I was thinking that someone could put the call in their plugin that runs on both the client and the server.

I don't have strong opinions on that so will just resolve. Let's say that user must explicitly do the client check.